labs/lab4/chapter4.md

```diff
@@ -60,7 +61,7 @@ And.. Your site is back! Fortunately wordpress seems to be designed such that it
 
 Finally, let's kill off these containers to prepare for the next section.
 ```bash
-$ docker rm -f wordpress mariadb
+$ sudo podman rm -f wordpress mariadb
```

Comment:
i think the order on this needs to be flipped (mariadb and then wordpres). I hit an error and I think it's related.